setup.py

```diff
@@ -22,7 +22,7 @@
     'argparse',
     'numpy',
     'requests',
-    'pandas>=0.21.0,!=0.24.0',
+    'pandas>=0.25.0',
```

Comment:
HI @zikolach, can you please do two things here:

First, make this line `pandas==0.23.4` so we can see the test failure explicitly

Second, update this line to `pandas>=0.21.0,!=0.24.0,!=0.23.4`